ethereumV2/contracts/factory/Factory.sol

```diff
@@ -52,6 +57,31 @@ contract Factory is OwnableContract {
         _;
     }
 
+    event HeartbeatSet(address indexed sender, uint256 heartbeat);
+
+    function setHeartbeat(uint256 newHeartbeat) external onlyCustodian returns (bool) {
+        // prevent possibility of overflow in validateProofOfReserves
+        require(newHeartbeat < block.timestamp, "invalid value for heartbeat");
+
+        // allow setting to 0 to fallback to DEFAULT_HEARTBEAT
+        heartbeat = newHeartbeat;
+
+        emit HeartbeatSet(msg.sender, getHeartbeat());
+
+        return true;
```

Comment:
Do we need to return a boolean? It seems like we revert if there's ever a problem, never return false. It would save a little gas to not return anything.

Reply:
Just copying how they use their existing setters, which also isn't apparent how the Boolean return value is used.